static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -0,0 +1,324 @@
+// @flow
+/* eslint-disable no-new */
```

Comment:
I think this is not required anymore?

Reply:
It is 😞  There are a few `new XYZ()` expressions used as side effects